### PR TITLE
Feature: VSDSPUB-615 Mongock

### PR DESCRIPTION
**Pull request by WLefever-Cegeka**
Mongock changeset for:
* adding `collectionName` to member and fragments
* imposing an order by adding `sequenceNr` to member (and the creation of the member sequence collection)

### CODE REVIEW

File: ldes-server-infra-mongo/mongock-changeset-3/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/infra/mongo/mongock/changeset3/entities/MemberSequenceEntityV1.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.infra.mongo.mongock.changeset3.entities;
```
Comment (sverholen) [inline ldes-server-infra-mongo/mongock-changeset-3/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/infra/mongo/mongock/changeset3/entities/MemberSequenceEntityV1.java:1]: overbodig ?

**Review by sverholen — APPROVED** [review]

File: ldes-server-infra-mongo/mongock-changeset-3/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Mongock Changeset 3
+
+The following configuration is needed to apply this changeset.
+
+This changeset relies on the ldes configuration to assign members and fragments to a collection and impose an order on the members. When more than one collection is configured,
+the migration does not know which version or timestamp to use. In this case it will use `UNDEFINED_COLLECTION` as collectionName. The reason why it uses this value instead of `null`, is because the collectionName is added as prefix to ids and properties. Adding `null` might lead to unexpected behaviour. Using the value `UNDEFINED_COLLECTION`, it will be easier to adapt or rollback if needed.
```
Comment (Tomvbe) [inline ldes-server-infra-mongo/mongock-changeset-3/README.md:6]: why do you refer to 'version or timestamp' ? Isn't that from a previous changeset?

**Review by Tomvbe — APPROVED** [review]

File: ldes-server-infra-mongo/mongock-changeset-3/README.md (hunk 2 above)
Comment (WLefever-Cegeka) [reply ldes-server-infra-mongo/mongock-changeset-3/README.md:6]: Haha, you got me! :-) Busted

File: ldes-server-infra-mongo/mongock-changeset-3/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/infra/mongo/mongock/changeset3/entities/MemberSequenceEntityV1.java (hunk 1 above)
Comment (WLefever-Cegeka) [reply ldes-server-infra-mongo/mongock-changeset-3/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/infra/mongo/mongock/changeset3/entities/MemberSequenceEntityV1.java:1]: No, the class is needed to give all the old members an index :-)

**Pull request merged**